src/test/resources/no/nav/tilleggsstonader/sak/vedtak/barnetilsyn/beregning/revurderFra/revurderFra.feature

```diff
@@ -18,8 +18,7 @@ Egenskap: Beregning - med revurderFra
 
     Når beregner med revurderFra=2024-02-15
 
-    Så forvent følgende beregningsresultat
-      | Måned | Dagsats | Antall dager | Utgift | Månedsbeløp |
+    Så forvent følgende feil: Kan ikke innvilge når det ikke finnes noen overlapp mellom målgruppe, aktivitet og utgifter etter revurder fra dato
```

Comment:
Lurer på om testen i utgangspunktet er en "dårlig oppsatt test" for det scenarioet "Skal ikke ta med perioder som slutter før måneden for revurderFra".

Burde man kanskje korrigere denne testen til være mer riktig til scenarioet, og at feilsituasjonen blir behandlet som en egen test?

Reply:
Kan gjøre det 👍